code/game/machinery/computer/Operating.dm

```diff
@@ -148,8 +155,12 @@
 /obj/machinery/computer/operating/machinery_process()
 	if(operable())
 		src.updateDialog()
+	if(src.stat & BROKEN)
```

Comment:
If it cannot be repaired from the broken state without rebuilding the machine, you can return `PROCESS_KILL` after the qdels so that it doesn't try to process anymore.

Reply:
Got it, and done! Thank you very much